fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java

```diff
@@ -284,6 +284,11 @@ private Void computeIcebergScanNode(Operator node, ExpressionContext context, Ta
             Statistics stats = GlobalStateMgr.getCurrentState().getMetadataMgr().getTableStatistics(
                     optimizerContext, catalogName, table, colRefToColumnMetaMap, null, node.getPredicate());
             context.setStatistics(stats);
+            if (node.isLogical()) {
```

Comment:
There is no need to check isLogical

Reply:
Just logical scan operator has filed hasUnknowColumn